### ISSUE TITLE
Please support rollup 2.x

### ISSUE THREAD

**Issue by lazka** [juliendargelos/rollup-plugin-emit-ejs#123]
```
npm WARN rollup-plugin-emit-ejs@3.0.0 requires a peer of rollup@^1.31.0 but none is installed. You must install peer dependencies yourself.
```

The plugin seems to work fine with rollup 2 so far though.

Thanks!

**Comment by juliendargelos** [comment]
Thanks for posting this issue, it has been fixed in v3.1.0

**Comment by lazka** [comment]
Awesome, thanks!
